hadoop-ozone/dist/src/main/smoketest/basic/links.robot

```diff
@@ -114,9 +114,13 @@ Key list passthrough
                         Should Contain              ${source_list}    key2
 
 Key delete passthrough
-                        Execute                     ozone sh key delete --skipTrash ${target}/link1/key2
-    ${source_list} =    Execute                     ozone sh key list ${source}/bucket1 | jq -r '.[].name'
-                        Should Not Contain          ${source_list}    key2
+                        Execute                     ozone sh key delete ${target}/link1/key2
+                        Execute                     ozone sh key list ${source}/bucket1 > bucket1keylist.json
```

Comment:
We should probably delete this file in the test teardown. Also saving to a string may be easier than the file since the key list output should not be that large. @adoroszlai what do you think?

Reply:
Good point.  In fact, before this patch the output was saved to string:

```
${source_list} =    Execute                     ozone sh key list ${source}/bucket1 | jq -r '.[].name'
```